lib/puppet/vendor/semantic/lib/semantic.rb

```diff
@@ -0,0 +1,5 @@
+require 'puppet/vendor/semantic_puppet/lib/semantic_puppet'
```

Comment:
> I wonder if its enough to just require 'puppet' to get the desired Rakefile behavior?

The desired behavior being - to have load `semantic_puppet` loaded? If I'm understanding correctly, I think that might work. But it would actually be a side-effect of the `semantic_puppet` lib being required by some other class/module that we require in `puppet.rb`, since the individual vendored libraries aren't actually required when we `require 'puppet/vendor'` - only added to the load path to enable them to be required.